src/test/java/com/epam/tc/hw2/ExerciseOneTest.java

```diff
@@ -0,0 +1,86 @@
+    private static String ID_FRAME_BUTTON = "frame-button";
+    private static String LIST_ITEMS_HEADER = "HOME\nCONTACT FORM\nSERVICE\nMETALS & COLORS";
+
+    @Test
+    public void exercise() {
+        //1. Open test site by URL_HOME_PAGE
+        driver.get(URL_HOME_PAGE);
+        //2. Assert Browser title "Home Page"
+        Assertions.assertThat(driver.getTitle())
+                .isEqualTo(HOME_PAGE);
+        //3. Perform login
+        WebElement loginDropdown = new WebDriverWait(driver, Duration.ofSeconds(5))
+                .until(ExpectedConditions.presenceOfElementLocated(By.xpath(XPATH_LOGIN_DROPDOWN)));
+        loginDropdown.click();
+        WebElement loginInput = driver.findElement(By.xpath(XPATH_LOGIN_INPUT));
+        loginInput.sendKeys(loginText);
+        WebElement passwordInput = driver.findElement(By.xpath(XPATH_PASSWORD_INPUT));
+        passwordInput.sendKeys(passwordText);
+        WebElement loginSubmit = driver.findElement(By.xpath(XPATH_LOGIN_SUBMIT));
+        loginSubmit.click();
+        WebElement logoutButton = driver.findElement(By.xpath(XPATH_LOGOUT_SUBMIT));
+        boolean logoutButtonIsDisplayed = logoutButton.isDisplayed();
+        Assertions.assertThat(logoutButtonIsDisplayed)
+                .isTrue();
+        //4. Assert Username is loggined
+        WebElement loginName = driver.findElement(By.id(ID_LOGIN_NAME));
+        Assertions.assertThat(loginName.getText())
+                .isEqualTo(LOGIN_ROMAN_IOVLEV);
+        //5. Assert that there are 4 items on the header section are displayed and they have proper texts
+        List<WebElement> itemsInHeader = driver.findElements(By.xpath(XPATH_ITEMS_IN_HEADER));
+        Assertions.assertThat(itemsInHeader.get(0).getText())
+                .isEqualTo(LIST_ITEMS_HEADER);
+        //6. Assert that there are 4 images on the Index Page and they are displayed
+        List<WebElement> listOfImages = driver.findElements(By.className("benefit-icon"));
+        Assertions.assertThat(listOfImages.size())
+                .isEqualTo(4);
+        //7. Assert that there are 4 texts on the Index Page under icons and they have proper text
+        List<WebElement> listOfTextUnderImages = driver.findElements(By.className("benefit-txt"));
+        Assertions.assertThat(listOfTextUnderImages.get(0).getText())
+                .isEqualTo("To include good practices\nand ideas from successful\nEPAM project");
+        Assertions.assertThat(listOfTextUnderImages.get(1).getText())
+                .isEqualTo("To be flexible and\ncustomizable");
+        Assertions.assertThat(listOfTextUnderImages.get(2).getText())
+                .isEqualTo("To be multiplatform");
+        Assertions.assertThat(listOfTextUnderImages.get(3).getText())
+                .isEqualTo("Already have good base\n(about 20 internal and\n"
+                        + "some external projects),\nwish to get more…");
```

Comment:
It's ok. But you can use map to get `List<String>` from `listOfTextUnderImages` and then compare it with expected string list.